### DISCUSSION TITLE
Doc Improvement: FastAPI Deployment Section

### DISCUSSION THREAD

**Discussion by stevemats** [fastapi/fastapi#6357] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
URL to issue: https://fastapi.tiangolo.com/deployment/manually/
```

## This doc section needs improvement: Not A Bug - Probably an ellipsis :)
 _Someone can rectify this while working on something else and push it all together_:
* Head over to: https://fastapi.tiangolo.com/deployment/manually/
* Now navigate to **_Run A Server Manually -Uvicorn_**, then to **_[Run the Server Program](https://fastapi.tiangolo.com/deployment/manually/)_**

![Run a Server Manually - Uvicorn - FastAPI](https://user-images.githubusercontent.com/30528167/161012475-6c3dd558-3f05-4b51-bd2d-a2ae1e84aa26.png)

### Operating System

Linux

### FastAPI Version

0.75.0

### Python Version

3.10.0

**Comment by jd-solanki** [comment]
What do you want to change in docs?

**Comment by stevemats** [comment]
> What do you want to change in docs?

Just a minor change:

Addition of "**Run**" in-between, '**_You can_**' and **_' then ..._**' as seen in the attached screenshot.

_I don't intend to change it. I was hoping if someone is working on another issue can as well rectify that word omission and ship the changes altogether._

**Comment by jd-solanki** [comment, marked as answer]
There's already PR https://github.com/tiangolo/fastapi/pull/4629

Thanks for reporting :)

**Comment by stevemats** [comment]
> There's already PR #4629
> 
> Thanks for reporting :)

Awesome 👍
